tools/azsdk-cli/Azure.Sdk.Tools.Cli/Models/Responses/VerifySetupResponse.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System.Text.Json.Serialization;
+using System.Text;
+namespace Azure.Sdk.Tools.Cli.Models;
+
+/// <summary>
+/// Response model for setup verification operations
+/// </summary>
+public class VerifySetupResponse : CommandResponse
+{
```

Comment:
@JennyPng Can you please add language as field int this response and update it. This will help us tag this tool call language level telemetry.